Packs/Cybereason/Scripts/CybereasonPreProcessingExample/CybereasonPreProcessingExample.py

```diff
@@ -2,26 +2,27 @@
 from CommonServerPython import *  # noqa: F401
 
 
-def main():
-    res = True
-    incident = demisto.incidents()[0]
+def get_guid_from_system_incident(incident: dict[str, Any]) -> str:
     malop_guid = ''
     for label in incident['labels']:
         if label['type'] == 'guidString':
             malop_guid = label['value']
+            break
+    return malop_guid
 
-    response = demisto.executeCommand('getIncidents', {'query': '-status:Closed and malopguid: {}'.format(malop_guid)})
-    malop_incident = response[0]['Contents']['data']
 
-    if malop_incident:
-        # Malop was already fetched - updating the relevant incident
-        res = False
-        malop_incident = malop_incident[0]
-        incident['id'] = malop_incident['id']
-        demisto.executeCommand('setIncident', incident)
+res = True
+incident = demisto.incidents()[0]
+malop_guid = get_guid_from_system_incident(incident)
 
-    demisto.results(res)
+response = demisto.executeCommand('getIncidents', {'query': '-status:Closed and malopguid: {}'.format(malop_guid)})
+malop_incident = response[0]['Contents']['data']
 
+if malop_incident:
+    # Malop was already fetched - updating the relevant incident
+    res = False
+    malop_incident = malop_incident[0]
+    incident['id'] = malop_incident['id']
+    demisto.executeCommand('setIncident', incident)
 
-if __name__ in ['__main__', '__builtin__', 'builtins']:
-    main()
```

Comment:
Why did we remove the main structure?

Reply:
It was not required, since whatever we are calling now everything is decalred outside of main. So we removed it.